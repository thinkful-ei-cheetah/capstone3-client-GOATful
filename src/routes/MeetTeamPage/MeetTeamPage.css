.landing-container {
  display: flex;
  flex-flow: column wrap;
  align-items: center;
}

.hero {
  background: url('../../img/sfr.jpg');
  background-size: cover;
  background-position: center;
  padding: 4rem 2rem;
  display: grid;
  height: 30vh;
}

.red {
  color: rgb(13, 22, 43);
  font-weight: 900;
  font-size: 28px;
}

.hero > * {
  color: white;
}

.verdi, .peter, .elan, .david, .kp, .click-through, .AB-testing, .about {
  max-width: 850px;
  margin: 1.5rem;
  padding: 2rem;
  border: 1px solid #E2E2E2;
  border-radius: .5rem;
}

.picSwap .goat{
  display: none; 
}

.picSwap:hover .goat{
  display: inline;
  position: absolute;
  top: 0;
  left: 0;
  z-index: 99;
  max-width: 850px;
}

.verdi > figure, .peter > figure, .elan > figure, .kp > figure, .david > figure, .click-through > figure, .AB-testing > figure, .about > figure {
  margin: auto;
  max-height: 600px;
  width: 600px;
  overflow: hidden;
  position: relative;
}

.peter > figure > img, .verdi > figure > img, .elan > figure > img, .kp > figure > img, .david >figure > img, .click-through > figure > img, .AB-testing > figure > img, .about > figure > img {
  max-width: 100%;
  max-height: 100%;
}

footer {
  background-color: rgba(0, 0, 0, 0.66);
  width: 100%;
  height: 60px;
  color: white;
  display: flex;
  justify-content: space-between;
  align-items: center;
}

footer > span {
  padding: .75em;
}

.btn-container {
  display: flex;
  justify-content: center;
  padding-bottom: 1rem;
}