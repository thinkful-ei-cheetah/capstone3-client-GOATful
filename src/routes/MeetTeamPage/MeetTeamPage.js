import React, { Component } from 'react'
import './MeetTeamPage.css'
import David from '../../img/david.jpeg'
import Kp from '../../img/KP.jpg'
import Elan from '../../img/elan.jpg'
import Verdi from '../../img/verdi.JPG'
import Peter from '../../img/peter.png'
import goat1 from '../../img/goat1.jpg'
import goat2 from '../../img/goat2.jpg'
import goat3 from '../../img/goat3.jpg'
import goat4 from '../../img/goat4.jpg'
import goat5 from '../../img/goat5.jpg'

class Team extends Component {
  componentDidMount() {
    console.log('I was mounted')
  }

  render() {
    window.scrollTo(0, 0);
    return (
      <>
        <section className='Teamhero'>
          <h1>
            Let your thumbnail do the talking.
        </h1>
          <div>
            <h2 className='red'>
              Meet the Laconic Team
          </h2>
          </div>
        </section>
        <div className='team-container'>
          <section className='david'>
            <figure className='picSwap'>
              <img src={David} alt="David K. Nordeen" />
              <img className='goat' src={goat1} alt="goat" />
            </figure>
            <h3>
              David K. Nordeen
            </h3>
            <p>
              Full stack developer web developer located in Scottsdale, AZ. I graduated from Drake University in 2016 with a Bachelor’s degree in marketing and entrepreneurial management. When i’m not programming, you can find me getting buckets at the local gym or watching a sci fi movie.
            </p>
          </section>

          <section className='elan'>
            <figure className='picSwap'>
              <img src={Elan} alt="Elan Green" />
              <img className='goat' src={goat2} alt="goat" />
            </figure>
            <h3>
              Elan Green
            </h3>
            <p>
              Full stack web developer currently living in Austin, TX. I am a classically trained cellist with 20 years of experience. I believe the intersection of music and technology is extremely powerful, and it is a great dream of mine to someday work for a music-focused tech company!
            </p>
          </section>

          <section className='kp'>
            <figure className='picSwap'>
              <img src={Kp} alt="Kristof-Pierre Cummings" />
              <img className='goat' src={goat3} alt="goat" />
            </figure>
            <h3>
              Kristof-Pierre Cummings
            </h3>
            <p>
              Full stack web developer located near Austin, TX. I love to camp, cook, and crack open a beer- especially in that order. Go Liverpool!
            </p>
          </section>
          <section className='verdi'>
            <figure className='picSwap'>
              <img src={Verdi} alt="Michael Verdi" />
              <img className='goat' src={goat4} alt="goat" />
            </figure>
            <h3>
              Michael Verdi
            </h3>
            <p>
              Full stack developer located in Tampa, FL. I'm a self taught engineer with three years experience in Ruby on Rails development working at both start ups and publicly traded companies.
            </p>
          </section>
          <section className='peter'>
            <figure className='picSwap'>
              <img src={Peter} alt="Peter Pae" />
              <img className='goat' src={goat5} alt="goat" />
            </figure>
            <h3>
              Peter Pae
            </h3>
            <p>
              Full stack developer and data/AI enthusiast from the DMV. I like to make keyboards, listen to music, and I smoke a mean brisket. I stay open minded and always seek new ways to improve my craft.
            </p>
          </section>
        </div>
      </>
    );
  }
}

export default Team