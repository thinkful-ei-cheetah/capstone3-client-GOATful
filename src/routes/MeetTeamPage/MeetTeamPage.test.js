import React from 'react';
import ReactDOM from 'react-dom';
import MeetTeamPage from './MeetTeamPage';
import renderer from 'react-test-renderer';

describe('MeetTeamPage Component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(< MeetTeamPage />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders the UI as expected', () => {
    const tree = renderer
      .create(<MeetTeamPage/>)
      .toJSON();
    expect(tree).toMatchSnapshot();  
  });
})
