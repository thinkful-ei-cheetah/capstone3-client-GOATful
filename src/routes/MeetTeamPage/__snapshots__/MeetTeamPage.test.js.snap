// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MeetTeamPage Component renders the UI as expected 1`] = `
Array [
  <section
    className="Teamhero"
  >
    <h1>
      Let your thumbnail do the talking.
    </h1>
    <div>
      <h2
        className="red"
      >
        Meet the Laconic Team
      </h2>
    </div>
  </section>,
  <div
    className="team-container"
  >
    <section
      className="david"
    >
      <figure
        className="picSwap"
      >
        <img
          alt="David K. Nordeen"
          src="david.jpeg"
        />
        <img
          alt="goat"
          className="goat"
          src="goat1.jpg"
        />
      </figure>
      <h3>
        David K. Nordeen
      </h3>
      <p>
        Full stack developer web developer located in Scottsdale, AZ. I graduated from Drake University in 2016 with a Bachelor’s degree in marketing and entrepreneurial management. When i’m not programming, you can find me getting buckets at the local gym or watching a sci fi movie.
      </p>
    </section>
    <section
      className="elan"
    >
      <figure
        className="picSwap"
      >
        <img
          alt="Elan Green"
          src="elan.jpg"
        />
        <img
          alt="goat"
          className="goat"
          src="goat2.jpg"
        />
      </figure>
      <h3>
        Elan Green
      </h3>
      <p>
        Full stack web developer currently living in Austin, TX. I am a classically trained cellist with 20 years of experience. I believe the intersection of music and technology is extremely powerful, and it is a great dream of mine to someday work for a music-focused tech company!
      </p>
    </section>
    <section
      className="kp"
    >
      <figure
        className="picSwap"
      >
        <img
          alt="Kristof-Pierre Cummings"
          src="KP.jpg"
        />
        <img
          alt="goat"
          className="goat"
          src="goat3.jpg"
        />
      </figure>
      <h3>
        Kristof-Pierre Cummings
      </h3>
      <p>
        Full stack web developer located near Austin, TX. I love to camp, cook, and crack open a beer- especially in that order. I think the future will be paved with AI and automation, and I want to help build it.
      </p>
    </section>
    <section
      className="verdi"
    >
      <figure
        className="picSwap"
      >
        <img
          alt="Michael Verdi"
          src="verdi.JPG"
        />
        <img
          alt="goat"
          className="goat"
          src="goat4.jpg"
        />
      </figure>
      <h3>
        Michael Verdi
      </h3>
      <p>
        Full stack developer located in Tampa, FL. I'm a self taught engineer with three years experience in Ruby on Rails development working at both start ups and publicly traded companies.
      </p>
    </section>
    <section
      className="peter"
    >
      <figure
        className="picSwap"
      >
        <img
          alt="Peter Pae"
          src="peter.png"
        />
        <img
          alt="goat"
          className="goat"
          src="goat5.jpg"
        />
      </figure>
      <h3>
        Peter Pae
      </h3>
      <p>
        Full stack developer and data/AI enthusiast from the DMV. I like to make keyboards, listen to music, and I smoke a mean brisket. I stay open minded and always seek new ways to improve my craft.
      </p>
    </section>
  </div>,
]
`;
